docs/manual/forms/examples/registration_attributes.rst

```diff
@@ -0,0 +1,95 @@
+.. _example_registration_attributes:
+
+=================================
+Form with registration attributes
+=================================
```

Comment:
Please do not make a generic "registration attributes" example, but a specific form-with-geometry/map example.